input/system/azure/system.go

```diff
@@ -0,0 +1,302 @@
+			for _, timeSeriesElement := range metric.TimeSeries {
+				for _, metricValue := range timeSeriesElement.Data {
+					if metricValue.Average != nil {
+						// value is total of 1min
+						diffedNetworkStats.TransmitThroughputBytesPerSecond = uint64(*metricValue.Average / 60)
+					}
+				}
+			}
+		} else if *metric.Name.Value == "network_bytes_ingress" {
+			for _, timeSeriesElement := range metric.TimeSeries {
+				for _, metricValue := range timeSeriesElement.Data {
+					if metricValue.Average != nil {
+						// value is total of 1min
+						diffedNetworkStats.ReceiveThroughputBytesPerSecond = uint64(*metricValue.Average / 60)
+					}
+				}
+			}
+		} else if *metric.Name.Value == "read_iops" {
+			for _, timeSeriesElement := range metric.TimeSeries {
+				for _, metricValue := range timeSeriesElement.Data {
+					if metricValue.Average != nil {
+						diffedDiskStats.ReadOperationsPerSecond = *metricValue.Average
+					}
+				}
+			}
+		} else if *metric.Name.Value == "write_iops" {
+			for _, timeSeriesElement := range metric.TimeSeries {
+				for _, metricValue := range timeSeriesElement.Data {
+					if metricValue.Average != nil {
+						diffedDiskStats.WriteOperationsPerSecond = *metricValue.Average
+					}
+				}
+			}
+		} else if *metric.Name.Value == "disk_queue_depth" {
+			for _, timeSeriesElement := range metric.TimeSeries {
+				for _, metricValue := range timeSeriesElement.Data {
+					if metricValue.Average != nil {
+						diffedDiskStats.AvgQueueSize = int32(*metricValue.Average)
+					}
+				}
+			}
+		} else if *metric.Name.Value == "read_throughput" {
+			for _, timeSeriesElement := range metric.TimeSeries {
+				for _, metricValue := range timeSeriesElement.Data {
+					if metricValue.Average != nil {
+						diffedDiskStats.BytesReadPerSecond = *metricValue.Average
+					}
+				}
+			}
+		} else if *metric.Name.Value == "write_throughput" {
+			for _, timeSeriesElement := range metric.TimeSeries {
+				for _, metricValue := range timeSeriesElement.Data {
+					if metricValue.Average != nil {
+						diffedDiskStats.BytesWrittenPerSecond = *metricValue.Average
+					}
+				}
+			}
+		} else if *metric.Name.Value == "storage_used" {
+			for _, timeSeriesElement := range metric.TimeSeries {
+				for _, metricValue := range timeSeriesElement.Data {
+					if metricValue.Average != nil {
+						if system.Info.Azure.StorageGB != 0 {
+							// Flexible Server
+							totalGB := uint64(system.Info.Azure.StorageGB)
+							system.DiskPartitions = make(state.DiskPartitionMap)
+							system.DiskPartitions["/"] = state.DiskPartition{
+								DiskName:   "default",
+								UsedBytes:  uint64(*metricValue.Average),
+								TotalBytes: totalGB * 1024 * 1024 * 1024,
+							}
+						} else if system.Info.Azure.CoordinatorStorageMB != 0 {
+							// Cosmos DB
+							// TODO: check if we need to sum up the node storage MB too
+							// (e.g. system.Info.Azure.NodeStorageMB * system.Info.Azure.NodeCount)
```

Comment:
@lfittl Do you know the answer for this, by any chance? I haven't tried it since making a node with Cosmos DB is really expensive 🙈

Reply:
My assumption is no, i.e. we don't need to consider the worker node storage as part of the coordinator's storage (they can be considered separate servers)